### DISCUSSION TITLE
pre-commit installation is broken

### DISCUSSION THREAD

**Discussion by apacha** [Homebrew/discussions#5028] [Casks] [answered]
### Output of `brew config`

```shell
n/a
```

### Output of `brew doctor`

```shell
n/a
```

### Description of issue

I've tried to install pre-commit on a fresh mac (apple silicon), but the python environment seems to be incomplete. I've tried reporting it [here](https://github.com/pre-commit/pre-commit/issues/3106) but they referred me back to homebrew. I'm not sure where to report this correctly, so I was hoping you could help me.

### Relevant casks

pre-commit

**Comment by branchv** [comment, marked as answer]
Thanks for the report, https://github.com/Homebrew/homebrew-core/pull/159180 will fix this
